src/mintpy/__main__.py

```diff
@@ -622,13 +674,17 @@ def main(*argv):
     # execute main tasks
     exit_code = EX_OK
     try:
-        return args.func(sys.argv[2:])
+        # NOTE: use the root logger to set the logging level
+        logging.getLogger().setLevel(args.loglevel)
+        log.debug("args: %s", args)
+
+        exit_code = args.func(sys.argv[2:])
     except Exception as exc:
         log.critical(
             "unexpected exception caught: {!r} {}".format(
                 type(exc).__name__, exc)
         )
-        log.debug("stacktrace:", exc_info=True)
+        log.exception("stacktrace:", exc_info=True)
```

Comment:
This is fine.
Please note, anyway, that, according to the documentation of the logging module, the `exc_info=True` is not strictly necessary for the `exception` method.

Reply:
Commented out the `exc_info` arg. Thanks.